_docker/drupal/drupal-filesystem/web/config/sync/user.role.content_admin.yml

```diff
@@ -16,6 +16,7 @@ permissions:
   - 'access site in maintenance mode'
   - 'access site reports'
   - 'add assembly entities'
+  - 'administer akamai'
```

Comment:
Is this a necessary permission for content_admin?

Reply:
Probably not, I may have gone a little "permissions checkbox happy" with that one.

Reply:
Resolved in commit 22eb3c1